src/xmipp/bindings/python/envs_DLTK/xmipp_DLTK_v1.0.yml

```diff
@@ -1,15 +1,15 @@
 #Protocols ussing the enviroment:
-#xmipp_deep_misalingment_detection
+#xmipp_deep_misalingment_detection, protocol_deep_center, protocol_deep_center_predict
 
 name: xmipp_DLTK_v1.0
 channels:
-  - anaconda
   - conda-forge
   - defaults
```

Comment:
Is defaults allowed? You removed it for the rest

Reply:
Thanks!